packages/query/src/index.ts

```diff
@@ -1002,17 +1021,14 @@ const generateQueryHook = async (
       'implementation',
     );
 
-    if (isVue(outputClient)) {
-      queryKeyProps = vueWrapTypeWithMaybeRef(queryKeyProps);
-    }
-
-    const routeString = isVue(outputClient)
-      ? getRouteAsArray(_route)
-      : `\`${route}\``;
+    const routeString = `\`${route}\``;
 
-    const queryKeyFn = `export const ${queryKeyFnName} = (${queryKeyProps}) => [${routeString}${
-      queryParams ? ', ...(params ? [params]: [])' : ''
-    }${body.implementation ? `, ${body.implementation}` : ''}] as const;`;
+    const queryKeyFn = `export const ${queryKeyFnName} = (${queryKeyProps}) => {
+    ${isVue(outputClient) ? vueUnRefParams(props) : ''}
```

Comment:
@Marcel-G is there any good reason why this was added? Please see https://github.com/anymaniax/orval/pull/931#issuecomment-1752355686 for context